### PR TITLE
auth pages mobile view

### LINKED ISSUE #75: Auth Pages Mobile View
- implement mobile views for auth pages (login, register, forgot password, reset password)

### PR DESCRIPTION
**Pull request by ctc-devops**
- implement mobile views for auth pages (login, register, forgot password, reset password)
closes #75

### CODE REVIEW

**Review by jeffcheema — COMMENTED** [review]
I couldn't find anything that could be improved upon, it looks all good to me :+1: 
Perhaps Megha might find some things that I may have overlooked.

File: src/components/ForgotPassword/ForgotPasswordForm.jsx [hunk 1]
```diff
@@ -69,7 +71,8 @@ const ForgotPasswordForm = () => {
   const onSubmit = async data => {
     try {
       await sendPasswordReset(data.email);
-      navigate('/login');
+      setOpenConfirmation(true);
+      // navigate('/login');
```
Comment (meghakak) [inline src/components/ForgotPassword/ForgotPasswordForm.jsx]: nit: please remove

File: src/common/CommonConfirmationPage/CommonConfirmationPage.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import FYALogoMobile from '../../assets/fya-logo-large.svg';
+import styles from './CommonConfirmationPage.module.css';
+
+const CommonConfirmationPage = ({ isOpen, children, showFYALogo }) => {
+  if (isOpen) {
+    return (
+      <div className={styles['confirmation-page']}>
+        {showFYALogo && <img src={FYALogo} className={styles.logo} alt="logo" />}
+        {showFYALogo && <img src={FYALogoMobile} className={styles['logo-mobile']} alt="logo" />}
+        {children}
```
Comment (meghakak) [inline src/common/CommonConfirmationPage/CommonConfirmationPage.jsx]: Can we add more common components to this CommonConfirmationPage? For example, the CheckCircleIcon is used in all the confirmation pages. We can also move the title and message Text components here and then just add props for the actual content for these (e.g. we would just pass in "Email Sent" and "A link to reset your password has been sent to your registered FYA email address" for the ForgotPasswordConfirmation).

Comment (meghakak) [reply src/common/CommonConfirmationPage/CommonConfirmationPage.jsx]: The Return to Login button can also be moved here if we are only using this CommonConfirmationPage for auth pages.

**Review by meghakak — CHANGES_REQUESTED** [review]
Awesome job on the mobile views + common confirmation page! The PR looks good for the most part, just a couple fixes for the CommonConfirmationPage. Could we also add the pictures (instead of the blue background) on the Figma for all the auth pages?

File: src/components/ForgotPassword/ForgotPasswordForm.jsx (hunk 1 above)
Comment (dgardezi) [reply src/components/ForgotPassword/ForgotPasswordForm.jsx]: Removed.

File: src/common/CommonConfirmationPage/CommonConfirmationPage.jsx (hunk 2 above)
Comment (dgardezi) [reply src/common/CommonConfirmationPage/CommonConfirmationPage.jsx]: I went ahead and pretty much added everything including the "Return to Login" button to the CommonConfirmationPage for now. I figured that if we needed to use this page elsewhere later we can just get rid of the button. Also I replaced the blue auth page backgrounds with their appropriate images but the styling is off, so I'll continue to work on that.

**Review by meghakak — COMMENTED** [review]
Thanks for adding the pictures and updating the common confirmation page! LGTM for now - will approve once the styling for images is done

**Review by meghakak — DISMISSED** [review]
LGTM

**Review by dgardezi — DISMISSED** [review]
LGTM! claude is my fav omfgggg **_#claudeboss_** jk megha is better #meghaboss

**Comment by meghakak** [comment]
> LGTM!

bro

**Comment by meghakak** [comment]
> LGTM!

### go away megha
## leave me alone duron

**Comment by dgardezi** [comment]
Hi sorry for another review request lol I just updated the images to the uncropped ones that Justine sent me

**Review by meghakak — DISMISSED** [review]
LGTM

File: src/common/CommonConfirmationPage/CommonConfirmationPage.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            <Text fontSize="48px" fontWeight="bold" className={styles['confirmation-title']}>
+              {confirmationTitle}
+            </Text>
+            <Text fontSize="18px" fontWeight="normal" className={styles['confirmation-text']}>
+              {confirmationText}
+            </Text>
+          </div>
+          <Button
+            className={styles['confirmation-button']}
+            onClick={() => returnToLogin()}
```
Comment (jeffcheema) [inline src/common/CommonConfirmationPage/CommonConfirmationPage.jsx]: I think you can do `onClick={returnToLogin}` , but I'm just nitpicking here :zany_face:

**Review by jeffcheema — CHANGES_REQUESTED** [review]
Just a small thing on the CommonConfirmationPage, but other than that it looks amazing!

File: src/common/CommonConfirmationPage/CommonConfirmationPage.jsx (hunk 3 above)
Comment (dgardezi) [reply src/common/CommonConfirmationPage/CommonConfirmationPage.jsx]: So this line actually occurs in multiple places throughout the app and all of the instances are written like the original code here. Should I update all of those instances of this or just leave it be?

Comment (meghakak) [reply src/common/CommonConfirmationPage/CommonConfirmationPage.jsx]: Good catch Gurneet! @Duron yes please, let's update all instances of this

Comment (dgardezi) [reply src/common/CommonConfirmationPage/CommonConfirmationPage.jsx]: Done!

**Review by meghakak — APPROVED** [review]
l g t m

**Review by jeffcheema — APPROVED** [review]
LGTM!

**Pull request merged**